lib/videoroom/meeting.ex

```diff
@@ -112,25 +82,16 @@ defmodule Videoroom.Meeting do
     end
   end
 
-  defp handle_room_error({:error, reason} = error, client, name) do
-    if String.contains?(reason, "does not exist") do
-      RoomRegistry.delete(name)
-      create_new_room(client, name)
-    else
-      error
-    end
-  end
-
   @impl true
   def handle_call(:add_peer, _from, state) do
     case Room.add_peer(state.client, state.room_id, Jellyfish.Peer.WebRTC) do
       {:ok, peer, token} ->
         Logger.info("Added peer #{peer.id}")
 
         timer = Process.send_after(self(), {:peer_timeout, peer.id}, state.peer_timeout)
-        peer_timers = Map.put(state.peer_timers, peer.id, timer)
+        state = put_in(state.peer_timers[peer.id], timer)
```

Comment:
```suggestion
        state = put_in(state,[:peer_timers,peer.id], timer)
```

Reply:
Does it change anything functionally, or just the looks? I prefer using  `put_in/2` vs `put_in/3`.

Reply:
Nope, it only changes how it looks. I mostly have seen `put_in/3`, but if you prefer `put_in/2`, it's ok.